src/components/media-player-track.js

```diff
@@ -151,6 +196,8 @@ MediaPlayerTrack.defaultProps = {
   onPlayStateChange: () => {},
   onProgress: () => {},
   posterImageUrl: null,
+  startTimeCode: null,
+  stopTimeCode: null,
```

Comment:
Small thing, but would be good to have consistent naming: we write "timecode" as one word in all other cases.